salsa_webqa/library/control_test.py

```diff
@@ -69,6 +69,8 @@ def gid(self, searched_id):
          If local execution parameter is "True", function will try to search for parameter in local configuration file.
          If such parameter is not found or there is an error while reading the file, server (default) configuration
          file will be used instead. """
+        if self.configs is None:
```

Comment:
Yes, having separate config for non-selenium tests seems reasonable to me. I had a look at existing config properties and I don't think we need to share variables between selenium & non-selenium tests.